sdk/face/ai-vision-face-rest/review/ai-vision-face.api.md

```diff
@@ -896,15 +896,6 @@ export interface CreateLivenessSessionContent {
     sendResultsToClient?: boolean;
 }
 
-// @public
-export interface CreateLivenessSessionContentForMultipart {
```

Comment:
Just confirm the model is removed from typespec?

Reply:
Yes I think so: https://github.com/Azure/azure-rest-api-specs/commit/969c68562c5c4850257c9242a7a64b73c15f110d
![image](https://github.com/Azure/azure-sdk-for-js/assets/38554834/7df5cb2e-b7ab-4296-aade-333416d40efc)

Reply:
Please ensure the typespec is merged before the SDK is merged.

Reply:
I see it's in `Azure/azure-rest-api-specs:main`. It's it good?

Reply:
yeah. keep on eye on new changes and our practice once the apiVersion is released it is not allowed to change the spec again.